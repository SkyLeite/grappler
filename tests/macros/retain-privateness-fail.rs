mod test {
    #[grappler::hook(signature = "AB CD")]
    fn foo() {}
}

pub fn main() {
    let _bar = test::foo;
}
