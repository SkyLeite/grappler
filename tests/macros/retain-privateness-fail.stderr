error[E0603]: constant `foo` is private
 --> tests/macros/retain-privateness-fail.rs:7:22
  |
7 |     let _bar = test::foo;
  |                      ^^^ private constant
  |
note: the constant `foo` is defined here
 --> tests/macros/retain-privateness-fail.rs:2:5
  |
2 |     #[grappler::hook(signature = "AB CD")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  = note: this error originates in the attribute macro `grappler::hook` (in Nightly builds, run with -Z macro-backtrace for more info)
